js/source/image_source.js

```diff
@@ -57,19 +56,27 @@ ImageSource.prototype = util.inherit(Evented, {
     onAdd: function(map) {
         this.map = map;
         if (this.image) {
-            this.createTile();
+            this.setCoordinates();
         }
     },
 
     /**
-     * Calculate which mercator tile is suitable for rendering the image in
-     * and create a buffer with the corner coordinates. These coordinates
-     * may be outside the tile, because raster tiles aren't clipped when rendering.
-     * @private
+     * Update image coordinates and rerender map
+     *
+     * @param {Array} coordinates lng, lat coordinates in order clockwise
+     * @returns {ImageSource} this
      */
-    createTile: function(cornerGeoCoords) {
+    setCoordinates: function(coordinates) {
+        this.coordinates = coordinates;
+
+        /**
+         * Calculate which mercator tile is suitable for rendering the image in
+         * and create a buffer with the corner coordinates. These coordinates
+         * may be outside the tile, because raster tiles aren't clipped when rendering.
+         */
```

Comment:
This should be a `//` style comment. `/**` style comments are reserved for semantic documentation blocks.